### PR TITLE
First cut of intro integration

### PR DESCRIPTION
**Pull request by clkao**
intro2

### CODE REVIEW

**Comment by okdistribute** [comment]
wow awesome idea!

File: app.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ var drive = hyperdrive(db)
 var speedometer = require('speedometer')
 var prettyBytes = require('pretty-bytes')
 var explorer = require('./')
+var intro = require('intro.js')
```
Comment (okdistribute) [inline app.js:13]: this can be `require('./intro')`

Comment (clkao) [inline app.js:13]: Is it a browserify magic? this is a crazy module named with `.js` under `node_modules/intro.js/`

Comment (okdistribute) [inline app.js:13]: ohhh weird. okay nevermind!

**Comment by okdistribute** [comment]
lgtm

**Pull request merged**